docs/docs/reference/developer_references/sandbox_reference/cli_wallet_reference.md

```diff
@@ -0,0 +1,154 @@
+---
+title: CLI Wallet
+tags: [sandbox, wallet, cli]
+---
+
+For development, it may be useful to deploy, transact, or create notes in a non-programmatic way. You can use Aztec's CLI Wallet for thing such as:
+
+- Deploying contracts
+- Sending transactions
+- Bridging L1 [Fee Juice](../../../protocol-specs/gas-and-fees/fee-juice.md) into Aztec
+- Pushing arbitrary [notes](../../../guides/developer_guides/smart_contracts/writing_contracts/notes/index.md) to your PXE
+- Creating [authwits](../../../guides/developer_guides/smart_contracts/writing_contracts/authwit.md)
+- Aliasing info and secrets for further usage
+
+:::info
+
+At any time, you can get an updated version of the existing commands and subcommands by adding `-h`. For example:
+
+```bash
+aztec-wallet create-account -h
+```
+
+:::
+
+## Aliases
+
+The CLI wallet makes extensive use of aliases, that is, when an address, artifact, secret, or other information is given a name that can be later used to reference it.
+
+Aliases have different types like `address` or `artifact` or `contract`. You can see a list of these types by running the help command `aztec-wallet alias -h`. You can then specify a type with the `:` character whenever needed. For example `accounts:master_yoda` or `artifacts:light_saber`.
+
+:::tip
+
+The wallet is smart enough to write to the `last` alias if it's likely that you use that same alias in the next command.
+
+It will also try and determine which type is expected. For example, if the alias `master_yoda` is an account, you don't need to prepend `account:` if, for example, you're deploying a contract.
```

Comment:
```suggestion
It will also try to determine which type is expected. For example, if the alias `master_yoda` is an account, you don't need to prepend `account:` if, for example, you're deploying a contract.
```